packages/phpstan-rules/src/Rules/CheckConstantStringValueFormatRule.php

```diff
@@ -71,7 +72,7 @@ public function process(Node $node, Scope $scope): array
                 continue;
             }
 
-            if (! $const->value instanceof String_) {
+            if (! $const->value instanceof String_ || $const->value instanceof Array_) {
```

Comment:
actually, `! $const->value instanceof String_` already handled it , so `$const->value instanceof Array_` is not needed.

Reply:
removed the `$const->value instanceof Array_` and the fixture `HasArrayClassContant` still works

Reply:
I have missed that. I'm sorry